hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -17,13 +18,12 @@
 # A dict of dicts organized by category, type and output file parm
 _HOUDINI_OUTPUTS = {
     # rops
-    hou.ropNodeTypeCategory(): {
-        "alembic": "filename",    # alembic cache
-        "comp": "copoutput",      # composite
-        "ifd": "vm_picture",      # mantra render node
-        "opengl": "picture",      # opengl render
-        "wren": "wr_picture",     # wren wireframe
-    },
+    hou.ropNodeTypeCategory(): [
+        "alembic",    # alembic cache
+        "geometry",  # geometry
```

Comment:
Just to align it with the alembic comment above

```suggestion
        "geometry",   # geometry
```